user-manual.md

```diff
@@ -0,0 +1,47 @@
+# A high level description of RecipeCart:
+This system is a website for uploading, searching, and rating recipes. Using our descriptive recipe tagging system (missing), users can specify dietary restrictions or preferences that match the recipes they wish to search for. Users can also leave ratings (missing) on recipes to let others know the average quality of the recipe.  
+
+# How to install the software:
+To install the software, there are separate requirements for the frontend and the backend. After cloning the repo to your local system, follow the steps below:
+- Frontend:
+    - Install Node.js (v18) from the Node website
+    - In the `/front-end/` directory, run `npm i` to install dependencies
+- Backend:
+    - 
+
+# How to run the software:
+After installing all the necessary dependencies, follow the steps below to run the software:
+- Frontend:
+    - In the `/front-end/` directory, run `npm run build`
+    - Run `npm run start` to launch the website
+    - Access the website at `http://localhost:8080`
+- Backend:
+    - 
```

Comment:
Okay to have this for now, but ideally for peer review, your user manual should be followable by non-tech user. Since your project is a website, installation + run should be just "visiting this URL in your (Chrome) browser"